app/controllers/idv/how_to_verify_controller.rb

```diff
@@ -55,13 +55,18 @@ def self.enabled?
         IdentityConfig.store.in_person_proofing_enabled
     end
 
+    def self.sp_ipp_enabled?(idv_session:)
+      idv_session.service_provider&.in_person_proofing_enabled
+    end
```

Comment:
I had split this out 10% to keep line 68-69 from being too unwieldy, and 90% because I thought it might be useful in stubbing out. It hasn't (yet?).

Reply:
I think it would be worth inlining below